package.json

```diff
@@ -26,7 +26,7 @@
   "scripts": {
     "start": "while(true) do node standupbot.js; sleep 5; done",
     "test": "mocha",
-    "migrations": "knex migrate:latest",
+    "migrations": "knex migrate:latest; knex seed:run",
```

Comment:
Are there any usecases where we don't want to seed the database? I'm nervous about forcing the seed when migrations are run.